app/serializers/group_serializer.rb

```diff
@@ -57,4 +57,8 @@ def has_discussions
   def has_multiple_admins
     object.admins.count > 1
   end
+
+  def cover_photo
```

Comment:
This mitigates the cover_photo N+1 some, but I'll want to go back at one point and find the right place for an eager load. I'm still thinking I really only want to load the cover photo if we're loading a page with a group theme, though.